src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -855,9 +854,8 @@ static int do_map(int argc, const char *argv[], Config *cfg)
     cout << cfg->devpath << std::endl;
```

Comment:
hi , 
after fork,  function global_init_postfork_start closed stdin and stdout， so this log message will not be printed?